### PR TITLE
V4.7.0

### PR DESCRIPTION
**Pull request by ocefpaf**
Alternative for #76

### CODE REVIEW

File: .ci_support/osx_mpinompi.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+CONDA_BUILD_SYSROOT:
+- /opt/MacOSX10.10.sdk
```
Comment (isuruf) [inline .ci_support/osx_mpinompi.yaml]: Where did this come from?

Comment (ocefpaf) [reply .ci_support/osx_mpinompi.yaml]: No idea. But somehow OS X is passing, the problem is with Linux :smile:

File: recipe/patches/strdup.patch [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+@@ -25,9 +25,9 @@
+ defined and missing types defined.
+ */
+ 
+-#ifndef HAVE_STRDUP
+-extern char* strdup(const char*);
+-#endif
++// #ifndef HAVE_STRDUP
++// extern char* strdup(const char*);
++// #endif
```
Comment (ocefpaf) [inline recipe/patches/strdup.patch]: @isuruf is this OK? It passes locally.

Comment (isuruf) [reply recipe/patches/strdup.patch]: Looking at the source, a better patch would be to add `#include "config.h"` at the top of this file.

**Comment by ocefpaf** [comment]
Thanks @isuruf! That patch worked like a charm!

**Comment by isuruf** [comment]
Can you send the patch upstream?

**Comment by ocefpaf** [comment]
> Can you send the patch upstream?

Working on it now.

**Comment by isuruf** [comment]
When sending it upstream move the new line 2 lines below

**Pull request merged**